cli/src/commands/parse.rs

```diff
@@ -0,0 +1,53 @@
+//! `phylum parse` command for lockfile parsing
+
+use std::io;
+use std::path::Path;
+
+use anyhow::Result;
+use phylum_types::types::package::{PackageDescriptor, PackageType};
+
+use super::lock_files;
+use crate::lockfiles::*;
```

Comment:
This certainly is super confusing. You've already suggested yourself to just move everything from `lock_files.rs` into this file. I think getting rid of these "duplicate" lock file modules with slightly different names would probably avoid some confusion in the future.

Reply:
I completely agree. But I may want to do that refactor into a separate PR to keep conflicts to a minimum.